lib/gauge_fix_ovr.cu

```diff
@@ -1368,11 +1368,11 @@ public:
         #else
           for ( int d = 0; d < 4; d++ ) {
             if ( !commDimPartitioned(d)) continue;
-            cudaMemcpyAsync(send[d], send_d[d], bytes[d], cudaMemcpyDeviceToHost, GFStream[d]);
+            qudaMemcpyAsyncNoTune(send[d], send_d[d], bytes[d], qudaMemcpyDeviceToHost, GFStream[d]);
```

Comment:
Why are these no tune variants?

Reply:
I made the assumption that if I saw a `cudaMemcpy` ass opposed to a `qudaMemcpy` then it was intentionally that was to skip tuning.

Reply:
The ones we explicitly do not want to tune are any calls that are made in `postTune` and `preTune` method, or helpers such as `backup` and `restore` that are called from those.

I believe all other calls to `cudaMemcpy`/`cudaMemcpyAsync` can be abstracted simply, as long as the `qudaMemcpy` / `qudaMemcpyAsync` variants check for active tuning (and don't tune their calls) then we should be good.

The only exception to this I believe are the calls to peer-to-peer memory copies where I don't believe we have the correct path set up for peer memory copies using the device interface (though I need to check that).  For those, a simple work around would be to call the run-time interface for those in the wrapper (and don't record anything if active tuning is happening).